### PR TITLE
Add native function invocation rule

### PR DESCRIPTION
**Pull request by alexmanno**
See https://github.com/facile-it/facile-coding-standard/issues/10

### CODE REVIEW

**Comment by alexmanno** [comment]
@thomasvargiu @Jean85

File: src/DefaultRules.php [hunk 1]
```diff
@@ -89,6 +89,7 @@
         'trim_array_spaces' => true,
         'unary_operator_spaces' => true,
         'whitespace_after_comma_in_array' => true,
+        'native_function_invocation' => true,
```
Comment (Jean85) [inline src/DefaultRules.php]: Please add it respecting alphabetical order.

**Comment by thomasvargiu** [comment]
I prefer to don’t add risky rules, they are dangerous. I’ve always removed them. I’d prefer they are optional.

**Comment by Jean85** [comment]
> @thomasvargiu So we can't add it?

I'll try to write down a PR for #5 to create a separate method to obtain risky rules, so maybe we can add it there.

**Comment by thomasvargiu** [comment]
Implemented in #16

**Pull request closed without merging**